controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/InternalStepRunner.java

```diff
@@ -548,6 +548,15 @@ private void logEndpoints(Map<ZoneId, List<Endpoint>> zoneEndpoints, DualLogger
                                                                                                            : " has config generation " + service.currentGeneration() + ", wanted is " + node.wantedConfigGeneration())));
     }
 
+
+    private String wantedPlatform(Node node) {
+        return node.wantedDockerImage().repository() + ":" + node.wantedVersion();
+    }
+
+    private String currentPlatform(Node node) {
+        return node.wantedDockerImage().repository() + ":" + node.wantedVersion();
```

Comment:
wanted -> current

Reply:
Fixed, and did your suggestion about only showing current if different from wanted